src/main/java/net/snowflake/client/core/IncidentUtil.java

```diff
@@ -62,169 +39,4 @@ public static String oneLiner(Throwable thrown) {
   public static String oneLiner(String prefix, Throwable thrown) {
     return prefix + " " + oneLiner(thrown);
   }
-
-  /**
-   * Dumps JVM metrics for this process.
-   *
-   * @param incidentId incident id
-   */
-  public static void dumpVmMetrics(String incidentId) {
```

Comment:
why to remove a public method?

Reply:
This method is never used anywhere

Reply:
if it's a public method, then it can be use anywhere else by an application or dependent object. So we need check with the team what is a deprecation process to remove a public method, otherwise it may break compilation of dependent application.

Reply:
thanks I wasn't aware of the driver being used this way. I removed the changes from this PR so we can just include the tests for now.